lib/localenv/clusterenv.go

```diff
@@ -41,14 +41,12 @@ import (
 // with all services initialized
 func (r *LocalEnvironment) NewClusterEnvironment(opts ...ClusterEnvironmentOption) (*ClusterEnvironment, error) {
 	client, _, err := httplib.GetClusterKubeClient(r.DNS.Addr())
-	if err != nil {
-		log.Errorf("Failed to create Kubernetes client: %v.",
-			trace.DebugReport(err))
+	if err != nil && !trace.IsNotFound(err) {
```

Comment:
Otherwise, the logs get spam for reasons that kubeconfig files are not available (which is true only after the installation).